src/chrome/content/rules/PIPNI.cz.xml

```diff
@@ -0,0 +1,29 @@
+<ruleset name="PIPNI s.r.o.">
+    <target host="pipni.cz" />
+    <target host="www.pipni.cz" />
+    <target host="webmail.pipni.cz" />
+    <target host="webftp.pipni.cz" />
+    <target host="mysql.pipni.cz" />
+    <target host="pgsql.pipni.cz" />
+    <target host="fbsql.pipni.cz" />
+    <target host="adminer.pipni.cz" />
+    <target host="forum.pipni.cz" />
+    <target host="monitoring.pipni.cz" />
+    <target host="manual.pipni.cz" />
+    <target host="web.pipni.cz" />
+    <target host="support.pipni.cz" />
+    <!--
+    Active Mixed-content:
+    <target host="stats.pipni.cz" />
+    -->
+
+    <securecookie host="^pipni\.cz$" name=".+" />
+    <securecookie host="^\.pipni\.cz$" name=".+" />
```

Comment:
No cookies of this form are used on stats.pipni.cz? Otherwise this would make them unavailable. It's hard to verify from here as the site requires authentication.